games/m.yaml

```
@@ -1533,6 +1533,25 @@
   video:
     youtube: _0rw36mA9mM
 
+- name: Mu-cade
+  originals:
+  - Geometry Wars
+  type: similar
+  url: 'https://www.asahi-net.or.jp/~cs8k-cyu/windows/mcd_e.html'
+  status: playable
+  content: open
+  langs:
+  - D
+  frameworks:
+  - SDL
+  - OpenGL
+  licenses:
+  - Custom
```

```suggestion
  - BSD2
```
according to readme in source